cmd/create.go

```diff
@@ -34,7 +34,16 @@ var createCmd = &cobra.Command{
 }
 
 func createCmdFunc(cmd *cobra.Command, args []string) {
-	remoteBackend, err := util.PromptForBackend()
+	// Get silent mode value
+	silentMode, err := cmd.Flags().GetBool("silent")
```

Comment:
Why not use viper?

Reply:
Oh, I forgot you could use viper for command line flags. That should be easy to change.